### PR TITLE
refactor ui

### PR DESCRIPTION
**Pull request by dworthen**
- migrate App to root
- migrate pages to root
- split pivot table of actions (prepare, select, etc) into separate pages
- Organize components, state, hooks into hierarchy based on use.

### CODE REVIEW

File: packages/webapp/src/App/Layout/Header/TitleBar/TitleBar.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			if (info) {
+				downloadAnchorRef.current.href = info.url
+				downloadAnchorRef.current.download = info.alias
+				downloadAnchorRef.current.click()
+			}
+		}
+	}, [downloadAnchorRef, onGetAllAssetsDownloadInfo])
+	return (
+		<Container align="center" justify="space-between">
+			<StyledLink to={Pages.Home.path}>{Pages.Home.name}</StyledLink>
```
Comment (rracanicci) [inline packages/webapp/src/App/Layout/Header/TitleBar/TitleBar.tsx:36]: Maybe we could put the `ProcessingProgress` component somewhere around here?

File: packages/webapp/src/components/Flexbox/Flexbox.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+		| 'space-evenly'
+	align?: 'stretch' | 'flex-start' | 'flex-end' | 'center' | 'baseline'
+	gap?: string
+	children?: React.ReactNode
+	className?: string
+	style?: React.CSSProperties
+}
+
+// @ts-expect-error Doesn't meet type constraint until later
+export const Flex: IFlex<FlexProps> = memo(function Flex({
```
Comment (rracanicci) [inline packages/webapp/src/components/Flexbox/Flexbox.tsx:32]: Maybe we should move this later on to a utility library. I can see this being useful to other projects.

File: packages/webapp/src/components/Pages/DataShowcasePage/DataShowcasePage.tsx [hunk 3]
```diff
@@ -1,162 +0,0 @@
...
-						<PivotItem
-							headerText={PipelineStep.Evaluate}
-							itemKey={PipelineStep.Evaluate}
-						/>
-						<PivotItem
-							headerText={PipelineStep.Navigate}
-							itemKey={PipelineStep.Navigate}
-						/>
-					</Pivot>
-					<ProcessingProgress />
```
Comment (rracanicci) [inline packages/webapp/src/components/Pages/DataShowcasePage/DataShowcasePage.tsx:141]: Maybe we should keep this as part of the title bar?

File: packages/webapp/src/pages/Home/HomePage.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+/*!
```
Comment (rracanicci) [inline packages/webapp/src/pages/Home/HomePage.tsx:1]: Is this component being used? If not, maybe we could remove it.

File: packages/webapp/src/pages/Navigate/DataNavigation/DataNavigation.tsx [hunk 5]
```diff
@@ -84,7 +83,9 @@ export const DataNavigation: React.FC = memo(function DataNavigation() {
 	const onClearSelectedAttributes = useOnClearSelectedAttributes(
 		setNewSelectedAttributesByColumn,
 	)
-	const onGoBack = useOnGoBack(setSelectedPipelineStep, PipelineStep.Evaluate)
```
Comment (rracanicci) [inline packages/webapp/src/pages/Navigate/DataNavigation/DataNavigation.tsx:87]: If this hook is not being used anymore, maybe we could completely remove it.

**Review by rracanicci — CHANGES_REQUESTED** [review]
It all looking great :) Just have some minor comments.

Also, I think I found a small bug. If after evaluating, you go to the navigate tab and the click the go back button. The synthesized/evaluate data goes away.

File: packages/webapp/src/App/Layout/Header/TitleBar/TitleBar.tsx (hunk 1 above)
Comment (dworthen) [reply packages/webapp/src/App/Layout/Header/TitleBar/TitleBar.tsx:37]: Placed the processing indicator in the title bar.

File: packages/webapp/src/pages/Home/HomePage.tsx (hunk 4 above)
Comment (dworthen) [reply packages/webapp/src/pages/Home/HomePage.tsx:1]: Removed.

**Pull request merged**